azurecompute/src/main/java/org/jclouds/azurecompute/options/AzureComputeTemplateOptions.java

```diff
@@ -89,6 +89,14 @@ public void copyTo(final TemplateOptions to) {
          if (storageAccountType.isPresent()) {
             eTo.storageAccountType(storageAccountType.get());
          }
+
+         final LoginCredentials credentials = new LoginCredentials.Builder().
+                 user(loginUser).
+                 password(loginPassword == null || !loginPassword.isPresent() ? null : loginPassword.get()).
+                 privateKey(loginPrivateKey == null || !loginPrivateKey.isPresent() ? null : loginPrivateKey.get()).
+                 authenticateSudo(authenticateSudo == null ? false : authenticateSudo).
+                 build();
+         eTo.overrideLoginCredentials(credentials);
```

Comment:
As you can see, I have opened [JCLOUDS-876](https://issues.apache.org/jira/browse/JCLOUDS-876) and provided a comment there; for the moment I have also reduced the temporary fix for this issue with last commit 00058df98c7d2e1c9a60ee006885d7610cb7bd3f